shared/desktop/renderer/index.js

```diff
@@ -53,6 +53,42 @@ function setupAvatar() {
   initAvatarLoad(loadUserImageMap, loadTeamImageMap)
 }
 
+var timeoutID
+
+// Consider us inactive after a minute of no input
+// for the purpose of marking chat messages read
+function setTimeoutListeners() {
```

Comment:
i think we can just do mousemove and keypress. we want to limit how many of these we have and we only need to support electron so we don't need mspointermove etc